domain/user/state/state.go

```diff
@@ -424,6 +427,45 @@ func AddUserWithPassword(
 	return errors.Trace(setPasswordHash(ctx, tx, uuid, passwordHash, salt))
 }
 
+// UpdateLastLogin updates the last login time for the user with the supplied
+// uuid. If the user does not exist an error that satisfies
+// usererrors.NotFound will be returned.
+func (st *State) UpdateLastLogin(ctx context.Context, uuid user.UUID) error {
+	db, err := st.DB()
+	if err != nil {
+		return errors.Annotate(err, "getting DB access")
+	}
+
+	updateLastLoginQuery := `
+UPDATE user_authentication
+SET last_login = $M.last_login
```

Comment:
This should be `DATE('now')`, that way this will be UTC. If you can't do this because of sqliar limitation, can you make the time UTC.

Reply:
done